src/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -382,10 +434,24 @@ void regclass_graph_Node(py::module m) {
         util::DictAttributeDeserializer dict_deserializer(attr_dict, variables);
         self->visit_attributes(dict_deserializer);
     });
-    node.def("set_arguments", [](const std::shared_ptr<ov::Node>& self, const ov::OutputVector& arguments) {
-        return self->set_arguments(arguments);
-    });
     node.def("validate", [](const std::shared_ptr<ov::Node>& self) {
+        Common::utils::deprecation_warning("validate",
+                                           "2024.4",
+                                           "Please use 'constructor_validate_and_infer_types' method instead.");
```

Comment:
Shall we tell anything about when it will be removed at all? For example, 2025.0

Reply:
It's already said that it will be removed in **2024.4**. I need just pass right version.